.github/config.yml

```diff
@@ -4,22 +4,27 @@
 
 # Comment to be posted to on first time issues
 newIssueWelcomeComment: |
-  Thanks for opening your first issue here! Please follow the issue template to help us help you 👍🎉😄
-  If you have screenshots to share demonstrating the issue, that's really helpful! 📸 You can [make a gif](https://www.cockos.com/licecap/) too!
+ Thanks for opening your first issue here! This space is [protected by our Code of Conduct](https://publiclab.org/conduct) - and we're here to help.
+ Please follow the issue template to help us help you 👍🎉😄
+ If you have screenshots to share demonstrating the issue, that's really helpful! 📸 You can [make a gif](https://www.cockos.com/licecap/) too!
+ Don't forget to join our [Gitter channel](https://gitter.im/publiclab/publiclab) for some brainstorming discussions. 
 # Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
 
 # Comment to be posted to on PRs from first time contributors in your repository
 newPRWelcomeComment: |
- Thanks for opening this pull request!
- There may be some errors, **but don't worry!** We're here to help! 👍🎉😄
+ Thanks for opening this pull request! This space is [protected by our Code of Conduct](https://publiclab.org/conduct).
+ `Dangerbot` will test out your code and reply in a bit with some pointers and requests.There may be some errors, **but don't worry!** We're here to help! 👍🎉😄
```

Comment:
I don't think we use danger bot, so probably you can remove this, instea add this
`There may be some errors, **but don't worry!** We're here to help! 👍🎉😄`